File: MPI/mpicheck/tests/functions/2.0/MPI_Comm_call_errhandler.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
        call mpi_comm_call_errhandler(var_0, var_1, var_2)
       call pmpi_comm_call_errhandler(var_0, var_1, var_2)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_Comm_call_errhandler.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Comm var_0;
    int var_1;
    int ret;
    /* calls */
    ret = MPI_Comm_call_errhandler(var_0, var_1);
    ret = PMPI_Comm_call_errhandler(var_0, var_1);
    return 0;
}
